hubspot/tasks.py

```diff
@@ -46,6 +48,25 @@ def sync_product_with_hubspot(product_id):
     response.raise_for_status()
 
 
+@app.task(bind=True)
+def sync_b2b_deal_with_hubspot(self, order_id):  # pylint: disable=unused-argument
+    """Send a sync-message to sync a b2b order with a hubspot deal"""
+    body = make_b2b_deal_sync_message(order_id)
+    response = send_hubspot_request("DEAL", HUBSPOT_SYNC_URL, "PUT", body=body)
+    response.raise_for_status()
+
+    index_tasks = celery.group([sync_b2b_product_with_hubspot.si(order_id)])
+    raise self.replace(celery.chain(index_tasks))
+
+
+@app.task
+def sync_b2b_product_with_hubspot(order_id):
+    """Send a sync-message to sync a line with a hubspot line item"""
```

Comment:
Can you add a unit test for this?  That should also end up covering the `make_b2b_product_sync_message` function